core/src/avm2/globals/flash/display/graphics.rs

```diff
@@ -72,8 +72,8 @@ pub fn begin_bitmap_fill<'gc>(
             height: bitmap.height() as u16,
         };
         let scale_matrix = Matrix::scale(
-            Fixed16::from_f64(bitmap.width as f64),
-            Fixed16::from_f64(bitmap.height as f64),
+            Fixed16::from_f64(Twips::TWIPS_PER_PIXEL as f64),
```

Comment:
I think this can be just `Twips::TWIPS_PER_PIXEL.into()`, reducing the `i32`-to-`f64` conversion and back.

Reply:
Thanks! Had to cast `as i16` first.